docs/interfaces/mx4/accelerometer.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Accelerometer
+tags:
+    - C61
+    - MX-4
+    - MX-V
+    - MX-4 T30
+    - MX-4 T30fr
+    - Linux 3.1
```

Comment:
This suggests Linux 3.1-specific information, but then you also mention Linux 6.1.

Reply:
i was thinking about the chapter ## For Linux 3.1 system. for linux 4 and up iio is used

Reply:
Ok, but my idea with tags is that they should be valid for all of the tags listed. If usage is different for some reason, then create a separate page where that specific information can be found. See e.g. mx4/leds

Reply:
I removed the Linux 3.1